todo_app/trello_config.py

```diff
@@ -0,0 +1,11 @@
+import os
+
+class TrelloStaticDataConfig:
```

Comment:
This is a really neat solution to this - one thing you might have issues with though in the next module is that this will set these variables when the file is _imported_ not when they're accessed, which may cause issues when we want our tests to set up fake environment variables before our code tries to access them

We should be able to solve that simply by adding a constructor (`__init__` function), and setting these all as `self.BOARD_ID` etc., then accessing them with `TrelloStaticDataConfig().FIELD` rather than `TrelloStaticDataConfig.FIELD`
(And yes, maybe we'd then want to drop the word static!)